code/controllers/configuration.dm

```diff
@@ -299,10 +311,30 @@
 
 				if("auto_cryo_ssd_mins")
 					config.auto_cryo_ssd_mins = text2num(value)
-
 				if("ssd_warning")
 					config.ssd_warning = 1
 
+				if("ipintel_email")
+					if(value != "ch@nge.me")

```

Comment:
reviewer's note: this is the default value added in the config file here

Reply:
Yes, and the reason its checked for is that accidentally enabling it with default value will result in your server IP quickly being banned by the webtool, which requires contacting them to reverse. So that's why there is an additional safety check that the value is not default.